entity/src/sbom_package_license.rs

```diff
@@ -0,0 +1,57 @@
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "sbom_package_license")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: Uuid,
+    pub sbom_id: Uuid,
+    pub node_id: String,
+    pub license_id: Uuid,
+    pub license_type: LicenseCategory,
```

Comment:
Only SPDX implicitly always provides the license category (i.e. though `licenseDeclared` and `licenseConcluded` fields) but CycloneDx introduced `acknowledgement` on in v1.6 as an optional field so I feel like this should change with:
```suggestion
    pub license_type: Option<LicenseCategory>,
```
@ctron I would like to hear your feedback on this proposed change.

Reply:
IIRC, we wanted to reach out to the spec authors would the absence of that information means.